jvm-runtime/ftl-runtime/pom.xml

```diff
@@ -400,42 +401,23 @@
                 </plugins>
             </build>
         </profile>
-        <!-- used to deploy artifacts to Maven Central via OSSRH (OSS Repository Hosting).
-             It configures the nexus-staging-maven-plugin for automated deployment and release,
-             and sets up the distribution management for both snapshot and release repositories.
-             Activate this profile when deploying open-source projects to Maven Central. -->
         <profile>
-            <!-- https://central.sonatype.org/publish/publish-maven/#deployment -->
-            <id>ossrh</id>
+            <id>central</id>
```

Comment:
Note that if you change this profile name you'll have to update the GH Action which calls "mvn deploy" to use the updated profile name